lib/config.py

```diff
@@ -10,6 +10,8 @@
 )
 sentinel_config_file = os.environ.get('SENTINEL_CONFIG', default_sentinel_config)
 sentinel_cfg = DashConfig.tokenize(sentinel_config_file)
+sentinel_version = 2
```

Comment:
Since this also impacts dashd side I'd rather discuss versioning sooner than later.